packages/tasit-link-wallet/README.md

```diff
@@ -2,14 +2,6 @@
 
 This package helps with onboarding a user by linking a dapp with a wallet of their choosing. This can be done either using ERC20 and ERC721 approve functions or using WalletConnect.
 
-**Important note**
-
-This onboarding-related child package is still in a _coming soon_ state. If you're checking out this repo for the purposes of considering whether the code is up to your standards, here are some completed portions of the code base to check out:
```

Comment:
We might want to keep this here until there are some more substantive features in this package

Reply:
Done.

Reply:
Thanks. Resolving.